make/modules/jdk.sctp/Lib.gmk

```diff
@@ -27,7 +27,15 @@
 
 include LibCommon.gmk
 
+SCTP_SUPPORTED=
```

Comment:
Ok, this should really match up with the check in Java.gmk. :-) So, "unix but not aix and not macos" basically meant "linux". :-) And here you indicate that freebsd is supported, while in the Java.gmk file the implication of excluding openbsd is that both freebsd and netbsd are supported. I can't say which one is right, but you should use the same check in both files. (Arguably, we could even lift the "SCTP_SUPPORTED" check to configure, but I don't propose we do it as part of the BSD port.)

Reply:
I like the idea of moving the SCTP_SUPPORTED check to the configure scripts.